docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -96,7 +96,9 @@ information that concern the file system:
   Total number of unallocated bytes in all file stores
 
 `fs.total.available_in_bytes`::
-  Total number of bytes available to this Java virtual machine on all file stores
+  Total number of bytes available to this Java virtual machine on all file stores.
+  Depending on OS or process level restrictions, this might appear less than fs.total.free_in_bytes.
```

Comment:
Can you put a backtick around `fs.total.free_in_bytes` so it shows as `<pre>`?

Reply:
sorry, will fix this.